tools/bazel/vendor.bzl

```diff
@@ -54,6 +75,13 @@ def _log_cargo_vendor(repository_ctx, result):
 vendor = repository_rule(
     doc = "A rule used to vendor the dependencies of a crate in the current workspace",
     attrs = {
+        "cargo_version": attr.string(
+            doc = "The version of cargo to use",
+            default = "1.51.0",
```

Comment:
Perhaps default this to `DEFAULT_RUST_VERSION` from `@rules_rust//rust:repositories.bzl`